Document/0x05c-Reverse-Engineering-and-Tampering.md

```diff
@@ -394,9 +394,9 @@ When reverse engineering apps, you'll often have access to the target app's rele
 
 To "convert" a release build into a debuggable build, you need to modify a flag in the app's manifest file. This modification breaks the code signature, so you'll also have to re-sign the altered APK archive.
 
-To re-sign, you first need a code-signing certificate. If you have built a project in Android Studio before, the IDE has already created a debug keystore and certificate in `$HOME/.android/debug.keystore`. The default password for this keystore is "android," and the key is called "androiddebugkey."
+To re-sign, you first need a code-signing certificate. If you have built a project in Android Studio before, the IDE has already created a debug keystore and certificate in `$HOME/.android/debug.keystore`. The default password for this KeyStore is "android," and the key is called "androiddebugkey."
 
-The standard Java distribution includes `keytool` for managing keystores and certificates. You can create your own signing certificate and key, then add it to the debug keystore:
+The standard Java distribution includes `keytool` for managing KeyStores and certificates. You can create your own signing certificate and key, then add it to the debug KeyStore:
```

Comment:
should it keep its capital when we are talking about the plural?

Reply:
I would say yes, but I can't immediately find a grammar rule to back me up ...